From 29df561ee4c875b7b71e8ad79120379a234e51a3 Mon Sep 17 00:00:00 2001
From: Robert Buchholz <rbu@goodpoint.de>
Date: Fri, 4 Mar 2016 16:32:53 +0100
Subject: [PATCH] Revert "Require setuptools>=1.0 in all packages that use the
 cryptography lib."

This reverts commit 55b63fca0dad0fae439bb3e453856987e37dea81.
RHEL7 does not come with setuptools >= 1.0, but only 0.9.8.
Since this dependency was only introduced as a workaround for a bug to
enable later versions of cryptography (in particular, cryptography 1.2),
we can safely revert this change until acme/letsencrypt actually uses
newer setuptools features.

Ref https://github.com/letsencrypt/letsencrypt/pull/2599
---

diff --git a/acme/setup.py b/acme/setup.py
index 0843288..5a77f8a 100644
--- a/acme/setup.py
+++ b/acme/setup.py
@@ -18,9 +18,7 @@ install_requires = [
     'pyrfc3339',
     'pytz',
     'requests',
-    # For pkg_resources. >=1.0 so pip resolves it to a version cryptography
-    # will tolerate; see #2599:
-    'setuptools>=1.0',
+    'setuptools',  # pkg_resources
     'six',
 ]
 
